backends-common/cassandra/pom.xml

```diff
@@ -134,4 +134,16 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
```

Comment:
didn't we have some issues with this parameter?

Reply:
Apparently, no.

Please not that I hand tested this manually, by running maven commands.